roles/container_runtime/tasks/extra_storage_setup.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Create file system on extra volume device
+  filesystem:
+    fstype: "{{ item.filesystem }}"
+    dev: "{{ item.device }}"
+  when:
+    - "'format' in item"
+    - item.format | bool
+    - inventory_hostname in groups[item.hosts|default('all')]
+  with_items: "{{ container_runtime_extra_storage|default([]) }}"
+
+
+- name: Create mount entry for extra volume
+  mount:
+    path: "{{ item.path }}"
+    src: "{{ item.device }}"
+    fstype: "{{ item.filesystem }}"
+    opts: "{{ item.options|default(omit) }}"
+    state: mounted
+  when:
+    - inventory_hostname in groups[item.hosts|default('all')]
```

Comment:
You can constrain this to a certain subset of hosts by placing the variable in the appropriate group in your inventory / group_vars.

EG, if you have nodes_west and nodes_east, you can put the dictionary in nodes_east group_vars only.